web/src/components/InputTextForm/InputTextForm.js

```diff
@@ -0,0 +1,34 @@
+import { getActiveClasses } from 'get-active-classes'
+import { TextField, FieldError } from '@redwoodjs/forms'
+import { useFormContext } from 'react-hook-form'
+
+const InputText = ({ type = 'text', className, name, validation }) => {
```

Comment:
Very light abstraction of individual text input that still working `with react-hoo-form`s.
![image](https://user-images.githubusercontent.com/29681384/101236955-1fb3dc80-3729-11eb-856e-456cd5b5a378.png)